style.css

```diff
@@ -0,0 +1,387 @@
+body{
+    margin: 0rem;
+    padding: 0rem;
+    font-family: sans-serif;
+}
+:root{
+    font-size: 10px;
+}
+.navbar{
+    height: 8.05rem;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 0.12rem solid #eee;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.logo{
+    height: 8.05rem;
+    border-right: 0.12rem solid #eeee;
+}
+.blink{
+    margin-left: 3.5rem;
+    color: #f8cb46;
+    font-size: 4.5rem;
+    font-family: inherit;
+    letter-spacing: -0.3rem;
+    font-weight: bold;
+}
+.it{
+    color: #54b226;
+    font-size: 4.5rem;
+    font-family: inherit;
+    letter-spacing: -0.3rem;
+    font-weight: bold;
+}
+.leftNav{
+    display: flex;
+    align-items: center;
+}
+.rightNav{
+    display: flex;
```

Comment:
`display: flex;
    align-items: center;`

Above style is duplicated, please create separate class for this style and reuse that class